File: Python/film.py
from produk import Produk

class Film(Produk):
    def __init__(self, id, judul, durasi, genre, sutradara, klasifikasi):
        super().__init__(id, judul, durasi)
        self.genre = genre
        self.sutradara = sutradara
        self.klasifikasi = klasifikasi

    def get_genre(self):
        return self.genre

    def get_sutradara(self):
        return self.sutradara

    def get_klasifikasi(self):
        return self.klasifikasi

    def set_genre(self, genre):
        self.genre = genre

    def set_sutradara(self, sutradara):
        self.sutradara = sutradara

    def set_klasifikasi(self, klasifikasi):
        self.klasifikasi = klasifikasi

File: Java/Film.java
public class Film extends Produk {
    private String genre;
    private String sutradara;
    private String klasifikasi;

    public Film(int id, String judul, int durasi, String genre, String sutradara, String klasifikasi){
        super(id, judul, durasi);
        this.genre = genre;
        this.sutradara = sutradara;
        this.klasifikasi = klasifikasi;
    }
    public String getGenre(){ return genre; }
    public String getSutradara(){ return sutradara; }
    public String getKlasifikasi(){ return klasifikasi; }
    public void setGenre(String genre){ this.genre = genre; }
    public void setSutradara(String sutradara){ this.sutradara = sutradara; }
    public void setKlasifikasi(String klasifikasi){ this.klasifikasi = klasifikasi; }
}